nav2_bringup/launch/nav2_params.yaml

```diff
@@ -95,7 +95,7 @@ local_costmap:
       robot_radius: 0.17
       inflation_layer.cost_scaling_factor: 3.0
       obstacle_layer:
-        enabled: False
+        enabled: True
```

Comment:
Minor observation: obstacle layer is already enabled by #762. In addition to changing this parameter, QoS setting for laser scan inside `ObstacleLayer` has to be set to `sensor_data`.

Reply:
Will rebase.